src/job-expanders.ts

```diff
@@ -22,6 +22,7 @@ const extendsRecurse = (gitlabData: any, jobName: string, jobData: any, parents:
 export function jobExtends(gitlabData: any) {
     for (const [jobName, jobData] of Object.entries<any>(gitlabData)) {
         if (Job.illegalJobNames.includes(jobName)) continue;
+        if (typeof jobData != "object") continue;
```

Comment:
What can the type then be?

Reply:
string, boolean, number see issue https://github.com/firecow/gitlab-ci-local/issues/484